### PR TITLE
Enable MGTwoLevelTransfer for FESystem

### PR DESCRIPTION
**Pull request by peterrum**
Only the last commit is relevant; the rest is part of #10966.

### CODE REVIEW

File: include/deal.II/multigrid/mg_transfer_global_coarsening.templates.h [hunk 1]
```diff
@@ -1615,7 +1632,9 @@ MGTwoLevelTransfer<dim, LinearAlgebra::distributed::Vector<Number>>::prolongate(
   LinearAlgebra::distributed::Vector<Number> &      dst,
   const LinearAlgebra::distributed::Vector<Number> &src) const
 {
-  const unsigned int vec_size = VectorizedArray<Number>::n_array_elements;
+  using VectorizedArrayType = VectorizedArray<Number>;
+
+  const unsigned int vec_size = VectorizedArrayType::n_array_elements;
```
Comment (kronbichler) [inline include/deal.II/multigrid/mg_transfer_global_coarsening.templates.h:1637]:
```suggestion
  const unsigned int n_lanes = VectorizedArrayType::size();
```
I think that is the common name in other files.

**Review by kronbichler — APPROVED** [review]

File: include/deal.II/multigrid/mg_transfer_global_coarsening.templates.h (hunk 1 above)
Comment (peterrum) [reply include/deal.II/multigrid/mg_transfer_global_coarsening.templates.h]: I have written these lines of code (to be precise: copied these lines) far before with consistently renamed this alias in the `matrix_free` folder ;)

I used the opportunity to rename the alias also in `MGTransferMatrixFree` !

Comment (peterrum) [reply include/deal.II/multigrid/mg_transfer_global_coarsening.templates.h]: Ready from my side after I have rebased with #10966.

**Comment by peterrum** [comment]
Ready from my side.

**Review by kronbichler — APPROVED** [review]

**Pull request merged**